documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -95,6 +95,13 @@ In this way the connector dynamically adjusts to changes in replica set membersh
 * link:https://docs.mongodb.com/manual/tutorial/remove-shards-from-cluster/[Shard removal]
 * link:https://docs.mongodb.com/manual/changeStreams/[Change Streams]
 
+// Type: concept
+// Title: Description of how the MongoDB connector uses the MongoDB read preference
+// ModuleID: description-of-how-the-mongodb-connector-uses-the-mongodb-read-preference
+[id="read-preference"]
+=== Read Preference
+Starting with Debezium 2.4 the MongoDB connector honors the link:https://www.mongodb.com/docs/manual/core/read-preference/[read preference] configured in either xref:mongodb-property-mongodb-connection-string[`mongodb.connection.string`] or xref:mongodb-property-mongodb-connection-string-shard-params[`mongodb.connection.string.shard.params`] for sharded clusters when xref:mongodb-property-mongodb-connection-mode[`mongodb.connection.mode`] is set to `replica_set`.
```

Comment:
I removed the statement about "Starting with Debezium 2.4", because I  think that it's not necessary. Users who are running earlier versions will read the docs for those versions, and those running 2.4 will read this version. The correct vehicle for alerting user to the change in behavior will be in the 2.4 release notes.

Reply:
I don't know bob. The information about addresses and whatnot is already in the description of `mognodb.connection.string`. What reading preference is is also a common knowledge for anyone using mongo db. So it seems a bit redundant to include it all. I wouldn't mind, however in this form I find the docs "hard to read" -- I just wanted to know how to set the read preference... this is 5+ paragraphs for something which could be 5 sentences (at least that's how I would feel when reading the docs).


Few additional problems

> It then applies the read preferences and other connection parameters that are specified in 
While technically true I think we should  mention that a connection string is constructed and these parameters are used as query parameters. 

> {prodname} tasks use a connection strings that is

Probably should have singular "connection string"

> The connector ignores any read preferences that are specified in the `mongodb.connection.string`. 

That's not true actually... the initial connection (used just to retrieve the topology info) will use it.

Reply:
cc @jpechane  @Naros  What is your opinion, guys? It seems to me that this is getting too complicated. However I might be 100% alone in this feeling.

Reply:
> I don't know bob. The information about addresses and whatnot is already in the description of `mognodb.connection.string`. What reading preference is is also a common knowledge for anyone using mongo db. So it seems a bit redundant to include it all. I wouldn't mind, however in this form I find the docs "hard to read" -- I just wanted to know how to set the read preference... 

OK, @jcechace. I get your point.  I think that I needed to talk my self through the concepts, and I included too much of that in the last version. Let me take one more stab at it to simplify things. Appreciate the input!

Reply:
@jcechace @roldanbob, so perhaps I'm a bit biased but I have seen lots of users who deploy Debezium who do not have a lot of database experience, and so while specific terminology or constructs may seem commonplace for us often are not for the layperson who is responsible for deploying the Debezium in the runtime environment.  

That said, I do agree that if we get overly verbose in such cases, this can have the opposite effect where users simply refuse to read the documentation because it is too wordy and even more confusing, so striking the right balance is crucial.  I don't know whether this is an opportunity to think about how admonitions can help clarify things for non-native database users in more depth without overly complicating or boring native database users.  Opinions?

Reply:
I agree, @Naros. It's difficult sometimes when I'm trying to describe something I'm unfamiliar with and don't have a good sense of what's common knowledge. 
In my [latest edit](https://github.com/debezium/debezium/pull/4654/files#r1262837732), I think that I moved the needle a bit. Did you guys see that version?

Reply:
The last iteration looks great, Bob. Thanks!